src/api/app/models/bs_request/find_for/base.rb

```diff
@@ -56,6 +57,11 @@ def ids
         @parameters[:ids]
       end
 
+      def limit
+        result = @parameters[:limit].to_i
+        result > 0 ? result : nil
```

Comment:
When does this limit get set as a negative number?

Reply:
c&p from the controller! When the user input is negative. And a negative limit would produce an invalid sql syntax 😭

Reply:
Do we really need to check that teh user hasn't entered a negative number? If so can we do that check with javascript on the form instead?

Reply:
I wouldn't check it if the output would then just be empty but it will crash with an ActiveRecord::StatementInvalid so I think we should handle it in the model, no?

Reply:
IMO this just adds a bit more complexity to the code for very little gain. What if the user enters a letter? Are we going to check for that too?

Reply:
Yo, because ``to_i`` will return 0 in this case :smile: 
So you mean dropping this commit completely?

Reply:
Yeah i would just get rid of this, or if you still want to check i would do it in the html/javascript.